src/style.css

```diff
@@ -1,3 +1,88 @@
+html {
+  --header-height: 60px;
+
+  margin: 0;
+  padding: 0;
+  font-family: Roboto, sans-serif;
+  font-size: 12px;
+}
+
+
 body {
   margin: 0;
+
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  width: 100%;
+
+  background-color: #fff;
+}
+
+.logo {
+  display: flex;
+  margin-left: 50px;
```

Comment:
instead of setting margin for children you should add padding to a parent (header). 
padding: 0 50px;